client/src/templates/layouts/Main.js

```diff
@@ -3,6 +3,8 @@ import { theme, mq } from "constants/theme";
 const { offWhite } = theme.colors;
 
 const Main = styled.main`
+  padding: 7rem 0rem 0rem 0rem;
```

Comment:
Please remove this completely.

Reply:
70 px is way too much padding...

Reply:
That's the current header's height. If I remove it, the content (feed) will be underneath the header

Reply:
No it won't happen. I removed it already on a previous PR. So let's just remove that padding property completely and let's merge this. Then we can double check,

Reply:
I've just double checked, with the css changes that added to make the header fixed. The main flex container slides underneath. Should I go and edit the padding on each page that has the header or should we just keep it with this main layout?

![screencapture-localhost-3000-feed-2020-06-18-16_00_04](https://user-images.githubusercontent.com/26049767/85080229-7fb7c600-b17d-11ea-9fcc-c8fca132108b.png)

Reply:
Yeah I see what you mean, then let's add some padding to that component specifically.

Reply:
I've just checked, it's probably not a good idea to do it specifically for each page affected as every page with the header is affected. eg.:

- /about-us
- /need-help
- /feed
- /nearest-hospital
- /offer-help
- /symptoms-check
...

Albeit for some of these pages the effect isn't that noticeable, but it's definitely shifted under. I think it makes sense to keep the padding with main until we decide to undertake reworking the layout templates.

Reply:
No I think you're right @jyclam 
I didn't want to suggest that, but seems a bit weird that we have a separate component `Main` and that it always gets the padding too, seems to me the padding should be set on the layout
maybe it doesn't even make sense to have the Main as a separate component, I think this was done a few weeks ago but some people have done weird things with the code that just got merged without proper reviews and we still have some technical debt there

Reply:
The strangest thing is that top padding on the main component was 0, so I am confused how now it is required?

Reply:
@idjevm I believe it's because `position:fixed` makes the header into a `positioned` element and a stacking context is also created. `Positioned` elements are displayed in front of `non-positioned` elements.

https://philipwalton.com/articles/what-no-one-told-you-about-z-index/

Reply:
Ahh that makes total sense, lol, let's get this merged then!